etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/model/dto/OrganizationOP.java

```diff
@@ -0,0 +1,77 @@
+package gov.cdc.etldatapipeline.changedata.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import gov.cdc.etldatapipeline.changedata.model.odse.DebeziumMetadata;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.*;
+
+@Data
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@EqualsAndHashCode
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrganizationOP extends DebeziumMetadata {
```

Comment:
We have to determine if we need the DebeziumMetadata. Now that we are using this java service as the processor.